changelog.md

```diff
@@ -2,6 +2,41 @@
 
 ## 2.4.0~rc1
 
+### Web Applications
+
+* Refactored application CSS and updated Source Interface design (#6322, #6362, #6315, #6419, #6429)
+* Updated code to remove Flask and Jinja2 deprecation warnings (#6245)
+* Fixed cases where string formatting was breaking translation lookups (#6368)
+* Simplified templates using jinja2 expression-statement extension (#6378)
+* Updated filesystem_id fields to be non-nullable (#6350)
+* Updated gnupg commands to use direct trust model (#6397)
+* Replaced potentially offensive terms in wordlists (#6402)
+* Added validation checks for configured vs. supported languages (#6406)
+
+### Journalist API
+
+* Updated datetime formatting to use custom encoder (#6260)
+
+### Operations
+
+* Added support for Tails 5.0 to Journalist and Admin Workstations (#6408, #6424)
+* Disabled fwupd timers (#6401)
+
+### Development
+
+* Fixed schema comparison tests (#6353) (drop?)
```

Comment:
I assume the `(drop?)` here is an oversight?

Reply:
yess dammit

Reply:
(fixed)